### PR TITLE
Update sphinx source to add additional schema.

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

File: sphinx/conf.py [hunk 1]
```diff
@@ -100,7 +100,7 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-html_logo = './Kraken.png'
+html_logo = './k2.png'
```
Comment (coffeepac) [inline sphinx/conf.py]: can we rename this png to be kraken.png?

**Comment by davidewatson** [comment]
e2e failed due to this:

> [Fail] [k8s.io] Projected [AfterEach] should be consumable in multiple volumes in a pod [Conformance] [Volume] 
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:283

It wasn't caused by this PR. This could be a kraken regression.

File: docs/_sources/index.rst.txt [hunk 2]
```diff
@@ -1,24 +1,29 @@
-.. k2 documentation master file, created by
+.. Kraken documentation master file, created by
```
Comment (coffeepac) [inline docs/_sources/index.rst.txt]: `kraken` not `Kraken`

File: docs/_sources/index.rst.txt [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
    sphinx-quickstart on Wed Jun 14 10:39:41 2017.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to k2's config file  documentation!
-==============================
+Welcome to Kraken's config file  documentation!
+===============================================
```
Comment (coffeepac) [inline docs/_sources/index.rst.txt]: `kraken`

File: docs/genindex.html [hunk 4]
```diff
@@ -6,12 +6,9 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    
-    <title>Index &#8212; k2 0.1 documentation</title>
-    
+    <title>Index &#8212; Kraken 0.1 documentation</title>
```
Comment (coffeepac) [inline docs/genindex.html]: `kraken`

File: sphinx/index.rst [hunk 5]
```diff
@@ -4,21 +4,26 @@
    contain the root `toctree` directive.
 
 Welcome to Kraken's config file  documentation!
```
Comment (coffeepac) [inline sphinx/index.rst]: `kraken`

File: docs/_sources/index.rst.txt (hunk 2 above)
Comment (davidewatson) [reply docs/_sources/index.rst.txt]: I think the APA actually says you should capitalize the first word of a sentence even if it is a proper noun which is normally lower case (cf. http://blog.apastyle.org/apastyle/2013/11/is-it-sometimes-okay-to-begin-a-sentence-with-a-lowercase-letter.html)

File: docs/_sources/index.rst.txt (hunk 3 above)
Comment (davidewatson) [reply docs/_sources/index.rst.txt]: Done.

File: docs/_sources/index.rst.txt (hunk 2 above)
Comment (coffeepac) [reply docs/_sources/index.rst.txt]: you win this time, gadget.

**Pull request merged**